third-party/realdds/include/realdds/dds-stream-profile.h

```diff
@@ -83,39 +48,52 @@ class dds_stream_base;
 
 class dds_stream_profile
 {
-    dds_stream_uid _uid;
-    dds_stream_format _format;
     int16_t _frequency;  // "Frames" per second
+    dds_stream_format _format;
 
     std::weak_ptr< dds_stream_base > _stream;
 
 public:
     virtual ~dds_stream_profile() {}
 
 protected:
-    dds_stream_profile( dds_stream_uid uid, dds_stream_format format, int16_t frequency )
-        : _uid( uid )
-        , _format( format )
+    dds_stream_profile( dds_stream_format format, int16_t frequency )
+        : _format( format )

```

Comment:
It is more efficient to initialize members by declaration order.

Reply:
AFAIK the init order is the order of decl, whatever order you write them in.